client/my-sites/plans/current-plan/index.jsx

```diff
@@ -69,6 +77,21 @@ class CurrentPlan extends Component {
 				' its professional features.' );
 		}
 
+		if ( plan === PLAN_JETPACK_PERSONAL || plan === PLAN_JETPACK_PERSONAL_MONTHLY ) {
+			tagLine = translate( 'Your data is being securely backed up and you have access to priority' +
```

Comment:
But let's ignore that. Seems it has been used like that consistently in other nested components.